src/plugins/discover/public/application/main/utils/fetch_text_based.ts

```diff
@@ -46,6 +47,15 @@ export function fetchTextBased(
       if (ast) {
         const contract = expressions.execute(ast, null, {
           inspectorAdapters,
+          titleForInspector: i18n.translate('discover.inspectorTextBasedRequestDataTitle', {
+            defaultMessage: 'Results',
+          }),
+          descriptionForInspector: i18n.translate(
+            'discover.inspectorTextBasedRequestDescriptionDocument',
+            {
+              defaultMessage: 'This request queries Elasticsearch using ES|QL to fetch results.',
+            }
+          ),
```

Comment:
Hi @amyjtechwriter,

Does it sound fine to you or how should I update it? That's for Inspector dropdown.

<img width="695" alt="Screenshot 2024-04-09 at 15 28 19" src="https://github.com/elastic/kibana/assets/1415710/0d2ba4f4-2142-433f-a265-833660eac62c">

Reply:
I think it sounds great!